app/email-incoming/index.ts

```diff
@@ -0,0 +1,172 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+
+import {
+  ListUsersInGroupCommand,
+  CognitoIdentityProviderClient,
+} from '@aws-sdk/client-cognito-identity-provider'
+import type { DynamoDB } from '@aws-sdk/client-dynamodb'
+import { DynamoDBAutoIncrement } from '@nasa-gcn/dynamodb-autoincrement'
+import type { DynamoDBDocument } from '@aws-sdk/lib-dynamodb'
+import { SendEmailCommand, SESv2Client } from '@aws-sdk/client-sesv2'
+import type { SNSEvent, SNSEventRecord } from 'aws-lambda'
+
+import { simpleParser } from 'mailparser'
+
+import {
+  subjectIsValid,
+  formatAuthor,
+  bodyIsValid,
+} from '../routes/circulars/circulars.lib'
+
+import { tables } from '@architect/functions'
+import { extractAttributeRequired, extractAttribute } from '~/lib/cognito'
+
+const client = new CognitoIdentityProviderClient({})
+const sesClient = new SESv2Client({})
```

Comment:
Rename these to be more descriptive:

```suggestion
const cognito = new CognitoIdentityProviderClient({})
const sesv2 = new SESv2Client({})
```

and update the rest of the code below.